sgx-traits/src/lib.rs

```diff
@@ -0,0 +1,94 @@
+// Copyright 2019 Red Hat, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::marker::PhantomData;
+use std::io::Result;
+
+use sgx_types::{secs::Secs, tcs::Tcs, sigstruct::SigStruct};
+use sgx_types::secinfo::{SecInfo, Flags as Permissions};
+use bitflags::bitflags;
+
+/// An offset reference with neither read nor write capabilities
```

Comment:
Was there more to this sentence?

Reply:
Nope. The first line of a doc comment is a sentence fragment with no period.